composer.json

```diff
@@ -10,6 +10,8 @@
         }
     ],
     "require": {
+        "php": "^7.1",
+        "ext-pdo": "*",
```

Comment:
Please add this to the `suggest` section with a proper message why this is suggested (like the commands that use this extension